1kproject/pay-expenses.js

```diff
@@ -10,7 +10,12 @@ const endpoint = `${process.env.API_URL}/graphql/v2/${process.env.API_KEY}`;
 
 const expensesQuery = gql`
   query {
-    expenses(account: { slug: "1kproject" }, limit: 1000, status: APPROVED) {
+    expenses(
+      account: { slug: "1kproject" }
+      limit: 1000
+      status: APPROVED
+      orderBy: { field: CREATED_AT, direction: ASC }
```

Comment:
Order by created at ASC to pay in the same order as created / entered in the sheet